data/korath/korath missions.txt

```diff
@@ -257,3 +257,53 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		not "Korath Boarding Response: active"
+		or
+			and
+				random < 5
+				"reputation: Korath" < -10
+				"reputation: Korath" >= -100
+			and
+				random < 10
+				"reputation: Korath" < -100
+				"reputation: Korath" >= -1000
+			and
+				random < 25
+				"reputation: Korath" < -1000
+				"reputation: Korath" >= -10000
+			and
+				random < 50
+				"reputation: Korath" < -10000
+
+			and
+				random < 10
+				"reputation: Korath (Civilian)" < 0
+				"reputation: Korath (Civilian)" >= -300
+			and
+				random < 25
+				"reputation: Korath (Civilian)" < -300
+				"reputation: Korath (Civilian)" >= -3000
+			and
+				random < 50
+				"reputation: Korath (Civilian)" < -3000
+	source
+		government Korath "Korath (Civilian)"
+		system Alcyone Almach "Alpha Hydri" Atik Durax Misam Polaris Sheratan
+	on offer
+		conversation
+			"An angry hiss comes over your radio, and you see a strange pulse along the hull of the Korath vessel. Your ship detects a powerful hyperwave emission followed by several capital ships jumping in. You had best finish your business with this ship before yours is reduced to a ball of superheated plasma."
```

Comment:
you do realize this will undock you from the Korath ship, right? mb mention that in the dialog

Reply:
I tested it. It does.

Reply:
Line 432 of conversationpanel clearly does this:
```cpp
else if(node != Conversation::ACCEPT && ship->GetGovernment()->IsEnemy()
				&& !ship->IsDestroyed() && ship->IsDisabled()
				&& ship->Position().Distance(player.Flagship()->Position()) <= 1.)
			GetUI()->Push(new BoardingPanel(player, ship));
```

Reply:
You're right, but now I'm worried. I will have to check my notes and see what happened in my earlier tests.

Reply:
I have updated the dialog.

Also, what happened before is that the "accept" was missing, so the ship was still connected, and the rescuers never arrived. That was earlier in my testing, before I remembered that "decline" is the default.

Reply:
`The pulse throws your ship off of theirs, but it appears you could board them again`